fiwareskuld/phase/phase3_delete.py

```diff
@@ -22,15 +22,15 @@
 # For those usages not covered by the Apache version 2.0 License please
 # contact with opensource@tid.es
 #
-from os import environ as env
-import os.path
 import cPickle as pickle
 import datetime
 
+from os import environ as env
+import os.path
```

Comment:
I am very interested in knowing why do you need to change the order of these imports?